src/LinAlg/hiopVectorRajaPar.cpp

```diff
@@ -852,7 +853,7 @@ double hiopVectorRajaPar::infnorm_local() const
   RAJA::forall< hiop_raja_exec >( RAJA::RangeSegment(0, n_local_),
     RAJA_LAMBDA(RAJA::Index_type i)
     {
-      norm.max(std::abs(data[i]));
+      norm.max(abs(data[i]));
```

Comment:
```suggestion
      norm.max(fabs(data[i]));
```
When using math functions from C library, it is `fabs`, otherwise it is absolute value of an integer; C++ std::abs is overloaded so you can pass floating point or integer argument to it.